guides/06_FAQs.md

```diff
@@ -0,0 +1,239 @@
+# Amazon Chime Javascript SDK - Frequently Asked Questions
+
+## General questions
+
+### What is the Amazon Chime SDK?
+
+The [Amazon Chime SDK](https://aws.amazon.com/chime/chime-sdk)is a set of real-time communications components that developers can use to quickly add audio calling, video calling, and screen sharing capabilities to their own web or mobile applications. Developers can leverage the same communication infrastructure and services that power Amazon Chime, an online meetings service from AWS, and deliver engaging experiences in their applications. 
+
+There are three client SDKs that can be used by applications based on the use case. Each of these repos have walkthroughs, and links to demos and sample code:
+
+Amazon Chime SDK for JavaScript https://github.com/aws/amazon-chime-sdk-js
```

Comment:
Fix the list layout. I'm previewing using this URL:
https://github.com/aws/amazon-chime-sdk-js/blob/2c6fd4a58a51087660e02381989a5fb834c7ece4/guides/06_FAQs.md